src/main/java/hometask3/Vector.java

```diff
@@ -0,0 +1,39 @@
+package hometask3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Vector {
+    private int x;
+    private int y;
+    private int z;
+
+    public Vector(int x, int y, int z) {
+        this.x = x;
+        this.y = y;
+        this.z = z;
+    }
+
+    public double getLength(int x, int y, int z) {
```

Comment:
This method shouldn't take any parameters, it should work with Vector's coordinates that already has x, y and z. Also, please delete the second getLength() method